core/commands/add.go

```diff
@@ -49,7 +49,7 @@ remains to be implemented.
 	},
 
 	Arguments: []cmds.Argument{
-		cmds.FileArg("path", true, true, "The path to a file to be added to IPFS").EnableRecursive().EnableStdin(),
```

Comment:
also, the text should ideally fit in one line. maybe:

```
The path to a file to be added to IPFS. If not set, uses stdin.
```
? not sure if `stdin` is too unfriendly.